lib/active_recall/models/deck.rb

```diff
@@ -55,19 +55,19 @@ def last
     end
 
     def untested
-      source_class.find(items.untested.pluck(:source_id))
+      source_class.where(id: items.untested.select(:source_id))
```

Comment:
Using `.select` rather than `.pluck` prevents even having to load an array of IDs and results in a single SQL query instead of two queries for each of the deck collection filters.